### PR TITLE
Add say/think functionality to looks blocks

### PR DESCRIPTION
**Pull request by paulkaplan**
### Proposed Changes

_Describe what this Pull Request does_

Adds say/think bubble functionality. Depends on https://github.com/LLK/scratch-render/pull/177 and https://github.com/LLK/scratch-render/pull/176.

### CODE REVIEW

File: src/engine/runtime.js [hunk 1]
```diff
@@ -280,22 +280,33 @@ class Runtime extends EventEmitter {
     }
 
     /**
-     * Event name for glowing the green flag
+     * Event name when threads start running.
+     * Used by the UI to indicate running status.
```
Comment (cwillisf) [inline src/engine/runtime.js:284]: 🥇

File: src/engine/runtime.js [hunk 2]
```diff
@@ -1228,6 +1242,16 @@ class Runtime extends EventEmitter {
     }
 
     /**
+     * Report that a new target has been created, possibly by cloning an existing target.
+     * @param {Target} newTarget - the newly created target.
+     * @param {Target} [sourceTarget] - the target used as a source for the new clone, if any.
+     * @fires Runtime#targetWasCreated
+     */
+    fireTargetWasRemoved (newTarget, sourceTarget) {
+        this.emit('targetWasRemoved', newTarget, sourceTarget);
```
Comment (cwillisf) [inline src/engine/runtime.js]: Looks like the comment & arg list need to be updated here

**Review by cwillisf — CHANGES_REQUESTED** [review]
Looks great! Just one C&P error :)

File: src/engine/runtime.js (hunk 2 above)
Comment (paulkaplan) [reply src/engine/runtime.js]: Good catch. Fixed in 
fa2db05

**Review by cwillisf — APPROVED** [review]
💬
:shipit:

**Pull request merged**